default.nix

```diff
@@ -1,22 +1,35 @@
-{ pkgs ? import (fetchTarball https://github.com/nixos/nixpkgs-channels/archive/cc1ae9f21b9e0ce998e706a3de1bad0b5259f22d.tar.gz) {} }:
-
-pkgs.stdenv.mkDerivation {
-  name = "hobbes";
-  src = ./.;
-
-  nativeBuildInputs = with pkgs; [
+{ pkgs ? import (fetchTarball { url="https://github.com/NixOs/nixpkgs/archive/a3ab47ec9067b5f9fccda506fc8641484c3d8e73.tar.gz";
+                                sha256="1f1q89ka73ksvl5hgbrf624x0niwm2rg3dzxiscir0bji6zvjy15";
+                              }) {} }:
+with pkgs;
+let
+  commonBuildInputs = [
     ccls
     cmake
     ninja
     meson
-    gcc8
-    llvm_6
     ncurses
     python27
     readline
     zlib
   ];
-
-  doCheck = false;
+  src = ./.;
+  doCheck = true;
```

Comment:
@thmzlt, @kthielen  - we should push these changes into [Nixpkgs](https://github.com/NixOS/nixpkgs/blob/f211631c1cb3e94828c7650b5d12c1e5a89e0e16/pkgs/development/tools/hobbes/default.nix#L25). I'll submit a PR to nixpkgs once this is merged, let me know if you do have any concern with this being reflected on Nixpkgs?

Reply:
:+1: